### PR TITLE
feat: page routing, collape result popup, save session token to cookies

### PR DESCRIPTION
**Pull request by daiwaid**
- add page routing (able to back/forward)
- add ability to collapse match result popup to the bottom of the screen
- saves session token (uuid) in cookies for up to 40 days

### CODE REVIEW

**Comment by daiwaid** [comment]
PR changes are now live at http://arena.libem.org

**Comment by zenodflow** [comment]
Very nice. The design is neat!

File: arena/src/Common.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { v4 as uuidv4 } from 'uuid'
+
+const baseURL = "http://127.0.0.1:8000"
```
Comment (zenodflow) [inline arena/src/Common.js:3]: We should support https / certificate in future; some browser does not open http tab nowadays.

**Review by zenodflow — APPROVED** [review]
Lgtm

**Pull request merged**